sdks/java/extensions/ml/src/main/java/org/apache/beam/sdk/extensions/ml/RecommendationAIWriteUserEvent.java

```diff
@@ -60,13 +58,13 @@ static Builder newBuilder() {
   }
 
   /** @return ID of Google Cloud project to be used for creating user events. */
-  public abstract @Nullable String projectId();
```

Comment:
Due to the change here.

In general it is possible breakage could happen elsewhere forcing previously nullable parameters to nonnull

Reply:
btw was this PR unreviewed?

Reply:
Oh sheesh. You are right. My mistake! I thought this was reviewed. I was kicking it for a long time on flakes.